templates/gotify/0/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  gotify:
+    image: cthit/gotify
+    environment:
+      GOTIFY_PORT: '8080'
+      GOTIFY_PRE-SHARED-KEY: SecretShareKey
+      GOTIFY_DEBUG-MODE: 'false'
+      GOTIFY_GOOGLE-MAIL.KEYFILE: gapps.json
+      GOTIFY_GOOGLE-MAIL.ADMIN-MAIL: admin@chalmers.it
+    stdin_open: true
+    volumes:
+    - /data/gotify/gapps.json:/app/gapps.json
```

Comment:
IRCC Rancher 1.X does not allow mounting individual files (everything is treated as a directory)

Reply:
Are you sure? This exact build (only the file mounted) is currently in use at starkiller.